app/Models/User.php

```diff
@@ -41,4 +41,5 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+    protected $guarded = ["is_admin"];
```

Comment:
khong the su dung guarded va fillable cung 1 model dc. check lai ly thuyet nha

Reply:
ready